src/OrchardCore.Modules/OrchardCore.Microsoft.Authentication/ViewModels/MicrosoftAccountSettingsViewModel.cs

```diff
@@ -14,5 +14,7 @@ public class MicrosoftAccountSettingsViewModel
         public string CallbackPath { get; set; }
 
         public bool SaveTokens { get; set; }
+
+        public bool CouldNotDecryptSettings { get; set; } = true;
```

Comment:
```suggestion
        public bool HasDecryptionError { get; set; }
```

Not sure why this one was set true by default?
Since you did it to all of them (great), I had to think about the appropriate name, for the now new pattern. 

Suggest `HasDecryptionError` might be better.

I think we should log the error at the time as well, as is done in the `Options` configuration classes - it may be that the error was on startup 3 weeks ago, but the user is only viewing now.

Reply:
I honestly don't know why I defaulted this one to true, other than I got distracted by something. I've pushed updates that address all of these issues.